### PR TITLE
Add CHANGLOG entries about raised OS requirements

### PR DESCRIPTION
**Pull request by daschuer**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -270,6 +270,8 @@
 * README: Recommend running buildenvs over sourcing them on Linux [#13071](https://github.com/mixxxdj/mixxx/pull/13071)
 * FindSndFile: Link mpg123 in static builds [#13087](https://github.com/mixxxdj/mixxx/pull/13087)
 * macOS packaging: Enable app sandbox in ad-hoc-packaged (i.e. non-notarized) bundles too [#12101](https://github.com/mixxxdj/mixxx/pull/12101)
+* Drop support for macOS versions earlier than 11, Windows 7 and 8.1 and Ubuntu 20.04 "Focal Fossa" and older.
```
Comment (JoergAtGithub) [inline CHANGELOG.md:273]: Windows 8 support and support for Windows 10 builds before 1809 is also dropped.
But I think, that of the mentioned versions, only Windows 7 has a significiant user base nowadays.

Comment (daschuer) [reply CHANGELOG.md]: done.

File: res/linux/org.mixxx.Mixxx.metainfo.xml [hunk 2]
```diff
@@ -804,6 +804,12 @@
    macOS packaging: Enable app sandbox in ad-hoc-packaged (i.e. non-notarized) bundles too
    #12101
   </li>
+  <li>
+   Drop support for macOS versions earlier than 11, Windows 7, Windows 10 versions up to 1803 and Ubuntu 20.04 "Focal Fossa"
```
Comment (JoergAtGithub) [inline res/linux/org.mixxx.Mixxx.metainfo.xml:808]:
```suggestion
   Drop support for macOS versions earlier than 11
   Drop support for Windows versions earlier than Windows 10 build 1809
   Drop support for Ubuntu versions earlier than 22.04
```
Let's write this in consistent style, not "earlier than" and "up to" mixed in one sentence.

**Comment by daschuer** [comment]
Done and updated with recent changes.

File: CHANGELOG.md [hunk 3]
```diff
@@ -94,6 +99,7 @@
 * Playlists: move tracks with Alt + Up/Down/PageUp/PageDown/Home/End
   [#13092](https://github.com/mixxxdj/mixxx/pull/13092)
   [#10826](https://github.com/mixxxdj/mixxx/issues/10826)
+  [#13098](https://github.com/mixxxdj/mixxx/pull/13098)
```
Comment (JoergAtGithub) [inline CHANGELOG.md:102]: This point is listed twice now

**Review by JoergAtGithub — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 3 above)
Comment (daschuer) [reply CHANGELOG.md:102]: I cannot confirm. Where is the second occurrence?

Comment (JoergAtGithub) [reply CHANGELOG.md:102]: grafik

**Comment by daschuer** [comment]
Done.

**Comment by JoergAtGithub** [comment]
LGTM! Thank you!

**Pull request merged**